reporting_reportrun.go

```diff
@@ -50,6 +50,7 @@ type ReportRunParameters struct {
 
 // ReportRun is the resource representing a report run.
 type ReportRun struct {
```

Comment:
damn it, this should be named `ReportingReportRun` instead.

Reply:
Doh, well that major version change is coming anyway. Should we put it on the list?

Reply:
Yep we have 2 other PRs pending for the next major so it'll be easy to release!